provisioning/service/src/Common/StringFormattingExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Provisioning.Service
+{
+    using System;
+    using System.Globalization;
+
+    public static class StringFormattingExtensions
```

Comment:
We are not doing localization - this should be removed.

Reply:
Agree, we need to revisit it.